### PR TITLE
Fix the week in Openverse posts

### LINKED ISSUE #327: "Week in Openverse" posts are broken
## Description

The 'a week in Openverse' posts on https://make.wordpress.org/openverse are broken. It looks like the last one was in March:

https://make.wordpress.org/openverse/tag/week-in-openverse/

We should fix these! I actually do not recall how or where the script for them was working. CC @dhruvkb, the original author.

### PR DESCRIPTION
**Pull request by zackkrida**
Switch to a `pnpm install` for the weekly update cron action.

This doesn't currently cache pnpm dependencies as we do on the frontend but I think that's okay given the once-per-week nature of this script.

## Fixes

Fixes #327 by @zackkrida 

Thanks to @AetherUnbound for the quick diagnostics here!

## Testing Instructions

Merge and try a manual run of the action, I suppose.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
I mainly copied this from our frontend node setup action, by the way.

File: .github/workflows/weekly_updates.yml [hunk 1]
```diff
@@ -26,7 +26,8 @@ jobs:
 
       - name: Install dependencies
         working-directory: ./automations/js
-        run: npm ci
+        shell: bash
+        run: "pnpm install"
```
Comment (AetherUnbound) [inline .github/workflows/weekly_updates.yml:30]: Why does `shell` need to be specified here? Can't we just run `pnpm install`?

**Review by AetherUnbound — APPROVED** [review]
LGTM!

File: .github/workflows/weekly_updates.yml (hunk 1 above)
Comment (zackkrida) [reply .github/workflows/weekly_updates.yml:30]: I'm honestly not sure. I recall there may have been a problem without it, as this shell configuration is present in our frontend repo action that behaves similarly to this one.

Comment (dhruvkb) [reply .github/workflows/weekly_updates.yml:30]: The `shell: bash` line is [optional](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell).

It's best to be clear if we are using some bash specific syntax but we aren't here.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**